src/core/variables/ColorVariable.ts

```diff
@@ -71,6 +71,16 @@ export class ColorVariable extends Variable implements IColorVariableParams {
   }
 
   /**
+   * Updates the selected value.
+   * @override
+   * @param {any} value The selected value.
+   */
+  updateValue(value: any): void {
```

Comment:
Yeah its an unfortunate limitation of typescript, that you cannot override a property (or getters/setters), only methods. So as a result I ended up breaking out an additional method that I can override in subclasses.